app/services/contents/builder.rb

```diff
@@ -30,13 +30,13 @@ def call
     def content_file_params_for(content:) # rubocop:disable Metrics/AbcSize
       cocina_object.structural.contains.flat_map do |file_set|
         file_set.structural.contains.map do |file|
-          filepath = file.filename
+          file_path = FilePath.new(filepath: file.filename)
```

Comment:
I find having `file_path` and `filepath` confusing. Is there some way to avoid this?

Reply:
I took another hack at it. Better? Worse? Different?